### PR TITLE
FAQ and Index changes

### PR DESCRIPTION
**Pull request by kylejonesdev**
This PR fills out the FAQ section with content I was able to source primarily from the SCN website and documentation.

It also streamlines the Home and Get Started pages into one page, emphasizing the importance of joining the community, but moving the instructions to do so to a separate page in the Community section. This makes it easier to link to the joining instructions from other pages, and makes the initial docs pages less intimidating.

Finally, the above addition of the joining instructions to the community section required a re-numbering of the pages already in the Community section.

I had to reconstruct the "Gemfile.lock" in order to get Jekyll working locally, but all of the changes to that file can probably be ignored.

### CODE REVIEW

File: faq/site.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: What is this site?
+parent: FAQ
+nav_order: 1
+---
+
+# About This Website
+The Seattle Community Network Docs website is the central hub for information about our community and networks. Here, we describe our infrastructure, how to set-up hardware and software, how you can start your own community network, our community rules, and more.
+
+This website is maintained by our volunteers, much like the rest of our services. This means you can help us improve it by adding missing information, clarifying confusing points, or even just fixing typos you notice while you’re reading. See CONTRIBUTE TO SCN DOCS to learn more about how you can contribute to this website.
```
Comment (infrared0) [inline faq/site.md]: let's make "contribute to scn docs" a link rather than all caps!

File: faq/how.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+nav_order: 4
+---
+
+# How Does the Seattle Community Network Work?
+
+The Seattle Community Network partners with the University of Washington to share free or low-cost internet access with areas of higher need.
+
+The Seattle Community Network functions by utilizing a portion of the University of Washington's internet bandwidth and sharing that access wirelessly with certain areas covered by the network. This is accomplished by using high gain wireless antennas to transmit data from the hardwired connection to a local node, which is then shared to nearby devices using the LTE (cell-phone data) standard. Individual users can then connect to this signal using an SCN-provided hardware, creating a local WiFi network to use for internet access in their home.
+
+The network is completely created, managed, and maintained by volunteers with a range of diverse skills in information technology and beyond. All infrastructure is paid for by generous donations from sponsors and the public. Speaking of which, why not VOLUNTEER or DONATE?
```
Comment (infrared0) [inline faq/how.md]: let's make "volunteer" and "donate" links rather than all caps!

File: faq/how.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+title: How does SCN Work?
+parent: FAQ
+nav_order: 4
+---
+
+# How Does the Seattle Community Network Work?
+
+The Seattle Community Network partners with the University of Washington to share free or low-cost internet access with areas of higher need.
+
+The Seattle Community Network functions by utilizing a portion of the University of Washington's internet bandwidth and sharing that access wirelessly with certain areas covered by the network. This is accomplished by using high gain wireless antennas to transmit data from the hardwired connection to a local node, which is then shared to nearby devices using the LTE (cell-phone data) standard. Individual users can then connect to this signal using an SCN-provided hardware, creating a local WiFi network to use for internet access in their home.
```
Comment (infrared0) [inline faq/how.md]: Would recommend replacing the 2nd paragraph with: 

"The Seattle Community Network (SCN) is a wireless Internet access network using 4G LTE and WiFi technologies, providing public access from partner locations such as libraries, schools, businesses, and community centers. The Internet connection at these sites is shared wirelessly to nearby devices using the 4G LTE (cell-phone) data standard, which can be used by certain phones and hotspots (also known as Customer Premises Equipment or CPE). Individual users can connect to this signal using SCN-provided (or other compatible) devices to create a local WiFi network in their home. Some of our installed sites utilize a portion of the University of Washington's internet bandwidth, sharing it out to further neighborhoods via point-to-point wireless links. Some sites use other upstream internet service providers (ISPs) such as Lumen."

**Review by infrared0 — CHANGES_REQUESTED** [review]

File: faq/site.md (hunk 1 above)
Comment (kylejonesdev) [reply faq/site.md]: Change completed in 849ddb8.

File: faq/how.md (hunk 2 above)
Comment (kylejonesdev) [reply faq/how.md]: Change completed in 849ddb8.

File: faq/how.md (hunk 3 above)
Comment (kylejonesdev) [reply faq/how.md]: Change completed in e36e9c2.

**Comment by kylejonesdev** [comment]
All changes made as requested. "All caps to links" issues were fixed in a commit already contained in the PR (see conversation above). Paragraph two of how.md modified as requested. Let me know if I missed anything.

**Pull request merged**